api/.docker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.6.2
+
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
+
+# Note that we want postgresql-client so 'manage.py dbshell' works.
+RUN apt-get update && \
+  apt-get install -y nodejs postgresql-client
```

Comment:
I can take out `postgresql-client` if you want, but I thought it'd be useful just so `manage.py dbshell` works.